server-pro/Dockerfile

```
@@ -108,6 +108,7 @@ ENV RSP_TESTUSER_PASSWD rstudio
 ENV RSP_TESTUSER_UID 10000
 ENV RSP_LAUNCHER true
 ENV RSP_LAUNCHER_TIMEOUT 10
+ENV DIAGNOSE_DIR /var/log
```

Let's do this to match with the logging spec https://app.tettra.co/teams/rstudio/pages/rstudio-logging-standard
```suggestion
ENV DIAGNOSE_DIR /var/log/rstudio
```